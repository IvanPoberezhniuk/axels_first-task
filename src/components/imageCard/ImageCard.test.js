import { unmountComponentAtNode } from 'react-dom';

import { ImageCard } from '../index';

const setUp = () => render(<ImageCard />);
let component;

beforeEach(() => {
  component = setUp();
});

afterEach(() => {
  unmountComponentAtNode(component);
  component.remove();
  component = null;
});

describe('ImageCard component', () => {
  it('SNAPSHOT', () => {
    expect(component).toMatchSnapshot();
  });
});

