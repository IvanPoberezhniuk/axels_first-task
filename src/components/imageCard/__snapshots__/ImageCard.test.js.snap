// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ImageCard component SNAPSHOT 1`] = `
<img
  alt="photo"
  class="sc-iBPRYJ kcuWgS rounded"
/>
`;
